### DISCUSSION TITLE
Git Performance Feedback

### DISCUSSION THREAD

**Discussion by fishcharlie** [blocklistproject/Lists#378] [General]
I'm curious to hear people's thoughts on the Git Performance for this project. I've noticed when doing certain things Git actions are **super** slow.

Is this something other people are experiencing? Would love to gather some feedback about how widespread this is and if others are running into it.

If a lot of people are running into it we can start looking into solutions for it.

**Comment by blocklistproject** [comment]
I have been receiving quite a few GH action errors lately.

**Reply by fishcharlie** [reply]
@blocklistproject Here is a recent build log. Looks like there are some formatting errors that it's catching. Not sure exactly what commit caused this. But in the future if we use PRs for any changes, and wait for the lint step to be successful, it'll be easier to catch these things before they make it into the project.

https://github.com/blocklistproject/Lists/runs/3316102285#step:4:4
